plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -324,20 +360,42 @@ open class KtlintPlugin : Plugin<Project> {
             description = "Runs ktlint on all kotlin sources in this project."
         }
 
+    private fun Project.getAndroidVariantMetaKtlintCheckTask(
+        variantName: String,
+        multiplatformTargetName: String? = null
+    ): Task {
+        val taskName = "ktlint${variantName.capitalize()}${multiplatformTargetName?.capitalize() ?: ""}Check"
+        return tasks.findByName(taskName) ?: task(taskName).apply {
+            group = VERIFICATION_GROUP
+            description = "Runs ktlint on all kotlin sources for android $variantName variant in this project."
+        }
+    }
+
     private fun Project.getMetaKtlintFormatTask(): Task = this.tasks.findByName(FORMAT_PARENT_TASK_NAME)
         ?: this.task(FORMAT_PARENT_TASK_NAME).apply {
             group = FORMATTING_GROUP
             description = "Runs the ktlint formatter on all kotlin sources in this project."
         }
 
+    private fun Project.getAndroidVariantMetaKtlintFormatTask(
+        variantName: String,
+        multiplatformTargetName: String? = null
+    ): Task {
+        val taskName = "ktlint${variantName.capitalize()}${multiplatformTargetName?.capitalize() ?: ""}Format"
+        return tasks.findByName(taskName) ?: task(taskName).apply {
+            group = FORMATTING_GROUP
+            description = "Runs ktlint formatter on all kotlin sources for android $variantName" +
+                " variant in this project."
+        }
+    }
+
     private fun setCheckTaskDependsOnKtlintCheckTask(project: Project, ktlintCheck: Task) {
         project.tasks.findByName("check")?.dependsOn(ktlintCheck)
     }
 
     /*
      * Helper functions used until Gradle Script Kotlin solidifies it's plugin API.
      */
-
```

Comment:
I think this is a line of documentation for all lower lines, so this newline gap was intentional.